service/api_logic/filter_manager/base_filter_manager.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy.orm import Query
+
+class BaseFilterManager:
+    FILTERS ={}
+
+    @classmethod
+    def apply_filters(cls, query: Query, filters) -> Query:
```

Comment:
what is "cls" and "filters" (I want to see ->  cls: SomeTypeIKnow, filters: SomeTypeIKnow)